### DISCUSSION TITLE
How to change the validation state after receiving a response from the server?

### DISCUSSION THREAD

**Discussion by Pahomk** [mdbootstrap/TW-Elements#2335] [Support from community]
How to change the validation state after receiving a response from the server?
I'm using the Vue JS 3 Composition API and Tailwind Elements 1.1.0

I am creating an authorization form with additional validation on the client side. For this I am using Tailwind Elements. 
For example, to check if a field value is an email address, I use the following attribute: `data-te-validation-ruleset="IsRequired|isEmail"`.

The client-side validation works fine.

Now I want that after submitting a form with an email address that was not previously registered and receiving a response from the server (for example: "no user with this email was found"), the email input field is also invalidated on the client side by applying the appropriate styles.

Thus. If a user entered an unregistered email address in the "Email" field (e.g. user123@gmail.com ) and clicked the "Login" button, then after the response from the server the Email field should become invalid on the client side as well.

There is an example in the Tailwind Elements [documentation][1] where you can programmatically set the field to an invalid state. But this example **DOESN'T WORK** if the "`data-te-validation-ruleset`" attribute is used additionally.

I beg everyone who is dealing with "Tailwind Elements", please help me! I spent more than 4 days and couldn't solve the problem.

  [1]: https://web.archive.org/web/20231208160606/https://tw-elements.com/docs/standard/forms/validation/#basic

**Comment by iprzybysz** [comment]
Hi [@Pahomk](https://github.com/Pahomk), I think that the easier way is to only manually control the validation like in the [Basic Example](https://tw-elements.com/docs/standard/forms/validation/#basic) without setting rulesets - btw. if you don't have access to this documentation, run it locally on your machine. Check out the [Local installation guide](https://tw-elements.com/docs/standard/getting-started/local-installation/) - Then depending on your response set the appropiate attribute and control your validation result.
